frontend/src/components/common/CircleBadge.tsx

```diff
@@ -0,0 +1,27 @@
+import { Circle } from "@chakra-ui/react";
+import React from "react";
+
+const CircleBadge = ({ count }: { count: number }): React.ReactElement => {
+  return (
+    <Circle
+      bgColor="white"
+      size={["50px"]}
```

Comment:
Does this have to be in an array format? Can we not have it just like "50px"

Reply:
Also, wondering if it would be good to make it variable based on the viewport, since we don't want the circle to get too large relative to the book size?

Reply:
Yep I'll make it responsive to viewport size.